content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -25,129 +25,129 @@
       </ul>
     </nav>
     <main>
-      <h1>Radio Group Example Using aria-activedescendant</h1>
+      <h1>aria-activedescendant を使用したラジオグループの例</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/962f6e771dccd4e3bbf8b83da717fb8d75b861da -->
 
       <section>
-        <h2>About This Example</h2>
+        <h2>この例について</h2>
         <img alt="" class="example-page-example-icon" src="../../../images/pattern-radio.svg">
         <p>
-          This example implements the features of the <a href="../radio-group-pattern.html">Radio Group Pattern</a> for two radio groups -- one for choosing a pizza crust and another for choosing a delivery method.
-          This implementation uses <code>aria-activedescendant</code> for informing assistive technologies which radio button has visual focus.
+          この例は、ピザの生地を選択するラジオグループと、配達方法を選択するラジオグループの 2 つのラジオグループに対して、<a href="../radio-group-pattern.html">ラジオグループパターン</a>の機能がどのように実装されているかを示しています。
```

Comment:
「どのように〜いるかを示しています」は原文にはなさそう
```suggestion
          この例は、ピザの生地を選択するラジオグループと、配達方法を選択するラジオグループの 2 つのラジオグループに対して、<a href="../radio-group-pattern.html">ラジオグループパターン</a>の機能を実装しています。
```